content/articles/momentum-based-gradient-optimizer-and-optimization-techniques-for-gradient-descent/index.md

```diff
@@ -0,0 +1,177 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /momentum-based-gradient-optimizer-and-optimization-techniques-for-gradient-descent/
+title: Gradient Descent Based Optimization Techniques 
+description: In this article, we consider Adaptive Optimization Algorithms and compare their convergence abilities from traditional gradient descent algorithms 
+author: bonface-ndolo
+date: 2021-11-19T00:00:00-18:00
+topics: [Languages]
+excerpt_separator: <!--more-->
+images:
+
+ - url: /engineering-education/momentum-based-gradient-optimizer-and-optimization-techniques-for-gradient-descent/hero.jpg
+   alt: Gradient Descent Based Optimization Techniques example image
+---
+
+Alternative models can be built using Gradient Descent in machine learning systems. An objective function specifies the level of inaccuracy that a Machine Learning model will have on a given dataset during the training phase.

```

Comment:
What do you mean by built using gradient descent? Gradient descent is used to minimize the loss function to learn the task appropriately.